### PR TITLE
Source Linnworks: re-enable in registry

### PR DESCRIPTION
**Pull request by ChristoGrab**
## What

The Linnworks source connector was disabled in our registries as part of the initiative to sunset deprecated connectors still using legacy state. This PR adds it back to the registries, as it was recently published with an updated CDK version in #34717.

## How

- Migrated dependency handling to Poetry
- Updated metadata
- Updated connector README

### CODE REVIEW

File: airbyte-integrations/connectors/source-linnworks/README.md [hunk 1]
```diff
@@ -7,51 +7,39 @@ For information about how to use this connector within Airbyte, see [the documen
...
 
 ```bash
-python main.py spec
-python main.py check --config secrets/config.json
-python main.py discover --config secrets/config.json
-python main.py read --config secrets/config.json --catalog integration_tests/configured_catalog.json
+poetry run source-linnworks spec
+poetry run source-linnworks check --config secrets/config.json
+poetry run source-linnworks discover --config secrets/config.json
+poetry run source-linnworks read --config secrets/config.json --catalog sample_files/configured_catalog.json
```
Comment (askarpets) [inline airbyte-integrations/connectors/source-linnworks/README.md:34]:
```suggestion
poetry run source-linnworks read --config secrets/config.json --catalog integration_tests/configured_catalog.json
```

File: airbyte-integrations/connectors/source-linnworks/pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+homepage = "https://airbyte.com"
+repository = "https://github.com/airbytehq/airbyte"
+[[tool.poetry.packages]]
+include = "source_linnworks"
+
+[tool.poetry.dependencies]
+python = "^3.9,<3.12"
+airbyte-cdk = "^0.61.2"
+vcrpy = "^6.0.0"
+pendulum = "==2.1.2"
```
Comment (askarpets) [inline airbyte-integrations/connectors/source-linnworks/pyproject.toml:22]: I believe `pendulum` is included as a part of `airbyte-cdk`, should it be removed?

**Review by askarpets — APPROVED** [review]

File: airbyte-integrations/connectors/source-linnworks/README.md (hunk 1 above)
Comment (ChristoGrab) [reply airbyte-integrations/connectors/source-linnworks/README.md]: Good catch, thanks!

File: airbyte-integrations/connectors/source-linnworks/pyproject.toml (hunk 2 above)
Comment (ChristoGrab) [reply airbyte-integrations/connectors/source-linnworks/pyproject.toml]: Didn't realize that, thanks for the heads up! Verified that it comes included with the CDK and updated the poetry/lock file

**Pull request merged**